### ISSUE TITLE
Create dynamic library without specifying main jar

### ISSUE THREAD

**Issue by jonvnieu** [excelsior-oss/excelsior-jet-maven-plugin#64]
Hi,

I would like to be able to create a dynamic library that does not contain any source code of its own ( i.e. there is no main jar artifact), but it just bundles several dependencies together. At the moment I can't do this easily, because the plugin expects a main jar to specified.

There are workarounds (e.g. using the maven-assembly-plugin to first create a jar-with-dependencies and then ignoring the project dependencies). But the solution would be more elegant if we could somehow specify that we are building a DLL that's purely based on libraries.

**Comment by pjBooms** [comment]
Would you like to you use `<packaging>pom</packaging>` for such a case?
If you use `<packaging>jar</packaging>`, Maven creates an empty jar if there is no sources (just dependencies) and jet plugin compiles that empty jar with all the dependencies into a DLL (that can be considered as another workaround).

**Comment by jonvnieu** [comment]
Yes, I was thinking about a scenario where <packaging>pom</packaging> is used.
For the time being I was already using the jar-packaging as a workaround.
